core/src/main/java/com/sequenceiq/distrox/v1/distrox/StackOperations.java

```diff
@@ -379,4 +383,9 @@ public CertificatesRotationV4Response rotateAutoTlsCertificates(@NotNull NameOrC
         LOGGER.debug("Starting cluster autotls certificates rotation: " + nameOrCrn);
         return clusterCommonService.rotateAutoTlsCertificates(nameOrCrn, workspaceId, certificatesRotationV4Request);
     }
+
+    public FlowIdentifier updateLoadBalancers(@NotNull NameOrCrn nameOrCrn, Long workspaceId) {
```

Comment:
Is there any reason why this is not part of `editByName(EnvironmentEditRequest)` rather than a different method altogether? Seems like the `EnvironmentEditRequest` already takes the `EnvironmentNetworkRequest` that contains info about access gateway related params.

Reply:
If so most of the classes in this PR may not be needed.

Reply:
As far as I can tell, the `editByName` logic just updates the environment as its stored in the database. It doesn't actually do things like change the created cloud resources to reflect the updated environment option, or create new resources. That's what I need to do. So I decided to create my own API call that would do a similar database update for the network, but then would also actually make the Stack service calls to create the load balancer and do all the configuration that needs to be done.